flow/storage/objectstorage/objectstorage_test.go

```diff
@@ -0,0 +1,123 @@
+package objectstorage
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"runtime"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/skydive-project/skydive/flow"
+	"github.com/skydive-project/skydive/storage/objectstorage/fake"
+)
+
+func generateFlowArray(count int) []*flow.Flow {
```

Comment:
please use real flow from pcap, like :
` 	flows := flowsFromPCAP(t, "pcaptraces/simple-tcpv4.pcap", layers.LinkTypeEthernet, nil)
`

Reply:
I don't really care for "real" flow values here. Why not just use empty flows like I'm doing?